diesel_derives/src/attr.rs

```diff
@@ -5,8 +5,8 @@ use util::*;
 
 #[derive(Debug)]
 pub struct Attr {
-    column_name: Option<syn::Ident>,
-    field_name: Option<syn::Ident>,
+    pub column_name: Option<syn::Ident>,
```

Comment:
Why do we need to make this public? Things should be using the `column_name()` method.

Reply:
Because otherwise the shorthand pattern for the destruction of the struct won't work. 
(In detail [this](https://github.com/diesel-rs/diesel/pull/1504/files#diff-cecdfd4f581eb6c37ace4cbea4fc8575R80) line would not be possible without making that thing public)
Maybe this should be moved to `Attr` as member funtcion?

Reply:
I don't see any shorthand destructuring of the struct. That line can and should be using the method form.

> Maybe this should be moved to Attr as member funtcion?

It already is there as a method. That is my point.